app/controllers/users_controller.rb

```diff
@@ -1,10 +1,9 @@
 class UsersController < ApplicationController
   def index
-    @users = User.all.order(created_at: :asc)
-    @current_user = current_user
+    @users = User.all
   end
```

Comment:
- [ ] This is what I got while trying to run your app with `rails s`. Kindly check your user controller for the error `undefined method "authenticate_user!"`.  Kindly check this [link](https://github.com/heartcombo/devise) for devise proper setup.
- [ ] You might also need to check your routes.

![error](https://github.com/uchexm/bloga/assets/26597432/79f5da6d-b8bb-4e7b-ab88-63fa4b8ee4ea)